components/Player/Pages/playerMatchesColumns.jsx

```diff
@@ -1,20 +1,18 @@
-import React from 'react';
 // import { Link } from 'react-router';
-import ActionDone from 'material-ui/svg-icons/action/done';
 import { transformations } from '../../../utility';
 
 export default [{
-  displayName: 'ID',
-  field: 'match_id',
-  width: 2,
-  sortFn: true,
-  displayFn: transformations.match_id,
-}, {
   displayName: 'Hero',
   field: 'hero_id',
-  width: 3,
+  width: 3.1,
   displayFn: transformations.hero_id,
 }, {
+  displayName: 'Id',
+  field: 'game_mode',
+  width: 1.8,
```

Comment:
I'm not sure if this is useful, so I would like your opinion, but I think we need to add a way for user to clear their sort. After merging this, there won't be a way to get the default ordering displayed unless you refresh the page or we change this from working ont he game_mode field to the id field (so that sort would work on id). The other option would be allowing there to be some sort of button users can click (which is intuitive) that would clear sorting. I'm not sure which is better (or even if we really care about it).